.github/workflows/main.yml

```diff
@@ -46,6 +42,7 @@ jobs:
           mkdir .output\Release\cleo\cleo_plugins
           mkdir .output\Release\cleo\cleo_saves
           mkdir .output\Release\cleo\cleo_text
+          mkdir .output\Release\cleo\cleo_sounds
```

Comment:
Sounds directory? I first time see that.
**Saves** and **text** are here because these are frequently used features and frequently asked by users how to use them.

Reply:
Yes, SAAS mod using it :)

Reply:
> Sounds directory? I first time see that. **Saves** and **text** are here because these are frequently used features and frequently asked by users how to use them.

Yes, SAAS mod using it :)

Reply:
Ok, but how is it CLEO related? There is not a single reference within the code to the cleo_sounds

Reply:
> Ok, but how is it CLEO related? There is not a single reference within the code to the cleo_sounds

Ok, i'm removing it then.